src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -9625,7 +9625,7 @@ void sofia_handle_sip_i_update(nua_t *nua, sofia_profile_t *profile, nua_handle_
 	}
 
 respond:
-	if (!sofia_test_flag(tech_pvt, TFLAG_BYE)) {
+	if (tech_pvt && !sofia_test_flag(tech_pvt, TFLAG_BYE)) {
```

Comment:
tech_pvt must be set otherwise will not respond to `UPDATE` request.

Reply:
Done